### PR TITLE
[stripe] Attach billing creator user id to Stripe customer

### PR DESCRIPTION
**Pull request by easyCZ**
## Related Issue(s)

* Part of https://github.com/gitpod-io/ops/issues/8192

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/usage/pkg/stripe/stripe.go [hunk 1]
```diff
@@ -23,8 +23,9 @@ import (
 )
 
 const (
-	AttributionIDMetadataKey     = "attributionId"
-	PreferredCurrencyMetadataKey = "preferredCurrency"
+	AttributionIDMetadataKey        = "attributionId"
+	PreferredCurrencyMetadataKey    = "preferredCurrency"
+	BillingCreaterUserIDMetadataKey = "billingCreatorUserId"
```
Comment (geropl) [inline components/usage/pkg/stripe/stripe.go:28]: Would be great to have a comment here what we use these for :cloud:

**Review by geropl — APPROVED** [review]
LGTM

/hold bc of [this comment](https://github.com/gitpod-io/gitpod/pull/16996/files#r1146264226)

File: components/usage/pkg/stripe/stripe.go (hunk 1 above)
Comment (easyCZ) [reply components/usage/pkg/stripe/stripe.go:28]: There are comments in the respective usages. I'll add something..

**Pull request merged**